datasets/iitb_en_hi/iitb_en_hi.py

```diff
@@ -0,0 +1,96 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""IIT Bombay English-Hindi Corpus"""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+import datasets
+
+
+_CITATION = """\
+Anoop Kunchukuttan, Pratik Mehta, Pushpak Bhattacharyya. \
+The IIT Bombay English-Hindi Parallel Corpus. Language Resources and Evaluation Conference. 2018.
+"""
+
+
+_DESCRIPTION = """\
+he IIT Bombay English-Hindi corpus contains parallel corpus for English-Hindi corpus collected from a variety of existing sources \
+and corpora developed at the Center for Indian Language Technology, IIT Bombay over the years.\
+The dataset contains 1,609,682 english to hindi translation pairs.
+"""
+
+_URLs = {
+    "train": "http://www.cfilt.iitb.ac.in/iitb_parallel/iitb_corpus_download/parallel.tgz",
+    "dev-test": "http://www.cfilt.iitb.ac.in/iitb_parallel/iitb_corpus_download/dev_test.tgz",
+}
+
+
+class IitbEnHi(datasets.GeneratorBasedBuilder):
+    """IIT Bombay English-Hindi Corpus"""
+
+    VERSION = datasets.Version("1.0.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="iitb_en_hi", description="IIT Bombay English-Hindi Corpus", version=datasets.Version("1.0.0")
+        )
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "english_text": datasets.Value("string"),
+                    "hindi_text": datasets.Value("string"),
```

Comment:
can you use the `Translation` feature type ?
you can see an example of usage in [flores](https://github.com/huggingface/datasets/blob/master/datasets/flores/flores.py)

Reply:
sure